sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v1/schema.yaml

```diff
@@ -2305,3 +2308,30 @@ fields:
 - name: startup_profile_selection_reason_first
   type: STRING
   mode: NULLABLE
+- name: first_document_id
+  type: STRING
+  mode: NULLABLE
+- name: partner_id
+  type: STRING
+  mode: NULLABLE
+- name: distribution_version
+  type: STRING
+  mode: NULLABLE
+- name: distributor
+  type: STRING
+  mode: NULLABLE
+- name: distributor_channel
+  type: STRING
+  mode: NULLABLE
+- name: env_build_platform_version
```

Comment:
Same comment about naming, which also differs from v2. It might be confusing for users.